src/ui/popup/theme/controls/mode.tsx

```diff
@@ -21,6 +32,13 @@ export default function Mode(props: {mode: string; onChange: (mode: string) => v
                     props.onChange(mode);
                 }}
             />
+            <span
```

Comment:
So I suggest you to make a
```jsx
<div class="mode-control-container">
  <DropDown />
  <span class="static-edit-button" />
</div>
```
`.mode-control-container` will have `position: relative`